src/root-redirect.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE HTML>
+<html lang="en-US">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="refresh" content="1;url=/#LATEST_VERSION#/docs/">
```

Comment:
ah, now I get it. Maybe lets change it back to the default (index.html) and add index.html to /docs folder? We were talking about having some 'intro' for docs anyway.